thirdparty/misc/polypartition.h

```diff
@@ -71,8 +71,8 @@ class TPPLPoly {
     return hole;
   }
 
-  void SetHole(bool hole) {
-    this->hole = hole;
+  void SetHole(bool p_hole) {
+    this->hole = p_hole;
   }
```

Comment:
It's really annoying that we still get not-super-important warnings emitted in thirdparty code despite disabling them.
We'd need to make sure those headers are treated as system headers by the compiler but that's a hassle in its own right.

Reply:
Shouldn't changes to third-party code be excluded from PRs that fix warnings?

Reply:
Well the problem is that if this warning is not fixed, the engine can't be built with `werror=yes`.

Reply:
I guess we can add `#pragma warning(disable : 4458)` defines around each `#include "thirdparty/misc/polypartition.h"` instead of changing it.

Reply:
There's 5 occurrences so I prefer patching the thirdparty code :)